lib/index.js

```diff
@@ -57,42 +57,63 @@ bedrock.events.on('bedrock.configure', function applyConfig() {
   }
 });
 
+function handleConfigLoadError(e, {configType}) {
+  // Only rethrow raw error in development to avoid leaking sensitive info.
```

Comment:
Style nit (single line (`//`) non-sentence, multiline full sentence + punctuation `(/* */)`).

```suggestion
  // only rethrow raw error in development to avoid leaking sensitive info
```